data/EventRow.css

```diff
@@ -1,10 +1,12 @@
 .event {
-    background: alpha(@colorAccent, 0.15);
+    background: alpha(@eventcolorAccent, 0.15);
     border-radius: 3px;
-    color: shade(@colorAccent, 0.65);
-    padding: 6px;
+    color: shade(@eventcolorAccent, 0.65);
+    padding: 10px;
 }
-
 .event .title {
     font-weight: 600;
 }
+.event .icon {
```

Comment:
You don't need to add a new class here. You can select with the element type: `.event image`